src/ORM/FieldType/DBField.php

```diff
@@ -520,11 +520,6 @@ public function debug(): string
 DBG;
     }
 
-    public function __toString(): string
-    {
-        return (string)$this->forTemplate();
-    }
```

Comment:
Exists on the superclass (which actually does exactly this anyway) so no need to deprecate.

Reply:
I left some comments about the use of ModelData::__toString() which I'm not really a fan of